changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Features
+
+- Add `ignite network request add-account` command.
```

Comment:
There was no discussion to reach a consensus but if you agree we are starting to add the link to the issue/PR in the changelog. We should discuss it though :).

```suggestion
- [#2955](https://github.com/ignite/cli/pull/2955) Add `ignite network request add-account` command.
```

Reply:
Oh, duplicate